docs/user/rstudio/ide/guide/tools/jobs.qmd

```diff
@@ -2,98 +2,101 @@
 
-A "background job" is an R script that runs in a separate, dedicated R session. You can run any R script in a separate session by:
+A "background job" is an R script that runs in a separate, dedicated R session. Any R script can be run in a separate session by:
 
-Selecting "Start Background Job" from the Background Job tab in the **Console pane**
+Selecting "Start Background Job" from the **Background Job** tab in the **Console**
 
-![](images/background-job-console-start.png){width="350"}
+![](images/background-job-console-start.png){fig-alt="Screenshot of the 'Start Background job' tab in the terminal pane."}
 
-Or from within a .R file by pulling down the Source menu and choosing *Source as Background Job*.
+Or from within a .R file by pulling down the Source menu and choosing **Source as Background Job**.
 
-![](images/source-job.png){fig-alt="Source script as local job" width="350"}
+![](images/source-job.png){fig-alt="Source script as background job" width="450"}
 
 Either method will give you some options for running your job.
 
-![](images/background-job-option.png "Dialog showing options for starting R script job"){alt="" width="450"}
+![](images/background-job-option.png){fig-alt="Dialog showing options for starting R script job"}
 
 By default, the job will run in a clean R session, and its temporary workspace will be discarded when the job is complete. This is the fastest and safest configuration, good for reproducible scripts that have no side effects.
 
 However, if you want to feed data from your current R session into the job, or have the job return data to your current R session, change the dialog options as follows:
 
-**Run job with copy of global environment**: If ticked, this option saves your global environment and loads it into the job's R session before it runs. This is useful because it will allow your job to see all the same variables you can see in the IDE. Note that this can be slow if you have large objects in your environment.
+-   **Run job with copy of global environment**: If ticked, this option saves your global environment and loads it into the job's R session before it runs. This is useful because it will allow your job to see all the same variables you can see in the IDE. Note that this can be slow if you have large objects in your environment.
+
+-   **Copy job results**: By default, the temporary workspace in which the job runs is not saved. If you'd like to import data from your job back into your R session, you have a couple of choices:
 
-**Copy job results**: By default, the temporary workspace in which the job runs is not saved. If you'd like to import data from your job back into your R session, you have a couple of choices:
+  -   **Global environment**: This places all the R objects your job creates back in your R session's global environment. Use this option with caution! The objects created by the job will overwrite, without a warning, any objects that have the same name in your environment.
```

Comment:
I may nest a callout here vs. having the "Use this option with caution" inline.

```suggestion
  -   **Global environment**: This places all the R objects your job creates back in your R session's global environment. 
  
  :::{.callout-caution}
  Use this option with caution! The objects created by the job will overwrite, without a warning, any objects that have the same name in your environment.
```

Reply:
added callout